x11-libs/tslib/tslib-1.0-r3.ebuild

```diff
@@ -8,8 +8,8 @@ inherit eutils toolchain-funcs autotools multilib-minimal
 
 PATCH_VER="5"
 DESCRIPTION="Touchscreen Access Library"
-HOMEPAGE="http://tslib.berlios.de/"
-SRC_URI="mirror://berlios/${PN}/${P}.tar.bz2
+HOMEPAGE="https://github.com/kergoth/tslib"
+SRC_URI="http://github.com/kergoth/${PN}/archive/${PV}.tar.gz
```

Comment:
Nope, can't do it like this:

```
Saving to: ‘/srv/nfs/common/distfiles/1.0.tar.gz’
```

I've mirrored the old tarball for you @ `http://dev.gentoo.org/~mgorny/dist/${P}.tar.bz2`. You know what to do :).